downstream/modules/platform/ref-hub-variables.adoc

```diff
@@ -0,0 +1,166 @@
+
+* provide an existing {HubNameMain} token as `automationhub_api_token`, or 
+
+* set `generate_automationhub_token` to `true` to generate a new token
+
+Generating a new token invalidates the existing token.
+| *`automationhub_authentication_backend`* a| This variable is not set by default. 
+Set it to `ldap` to use LDAP authentication. 
+
+When this is set to `‘ldap’`, you must also set the following variables:
+
+* `automationhub_ldap_server_uri`
+* `automationhub_ldap_bind_dn`
+* `automationhub_ldap_bind_password`
+* `automationhub_ldap_user_search_base_dn`
+* `automationhub_ldap_group_seach_base_dn`
+
+| *`automationhub_auto_sign_collections`* | If a collection signing service is enabled, collections are not signed automatically by default. 
+
+Setting this parameter to `true` signs them by default.
+
+Default = `false`.
+| *`automationhub_backup_collections`* | _Optional_
+
+{HubNameMain} provides artifacts in `/var/lib/pulp`. 
+{ControllerName} automatically backs up the artifacts by default. 
+
+You can also set `automationhub_backup_collections = false` and the backup/restore process does not then backup or restore `/var/lib/pulp`.
+
+Default = `'true'`
+| *`automationhub_collection_signing_service_key`* | If a collection signing service is enabled, you must provide this variable to ensure collections can be properly signed. 
+
+`/absolute/path/to/key/to/sign`
+| *`automationhub_collection_signing_service_script`* | If a collection signing service is enabled, you must provide this variable to ensure collections can be properly signed. 
+
+`/absolute/path/to/script/that/signs`
+| *`automationhub_create_default_collection_signing_service`* | The default install does not create a signing service. 
+If set to `true` a signing service is created.
+
+Default = `false`
+| *`automationhub_disable_hsts`* | The default install deploys a TLS enabled {HubNameMain}.
+use if {HubName} is deployed with _HTTP Strict Transport Security_ (HSTS) web-security policy enabled.
+Unless specified otherwise, the HSTS web-security policy mechanism is enabled. 
+This setting allows you to disable it if required.
+
+Default = `false`
+| *`automationhub_disable_https`* | _Optional_
+
+If {HubNameMain} is deployed with HTTPS enabled.
+
+Default = `false`.
+| *`automationhub_enable_api_access_log`* | When set to `true`, creates a log file at `/var/log/galaxy_api_access.log` that logs all user actions made to the platform, including their username and IP address.
+
+Default = `false`.
+| *`automationhub_importer_settings`* | _Optional_ 
+
+Dictionary of setting to pass to galaxy-importer.
+
+At import time collections can go through a series of checks.
+
+Behavior is driven by `galaxy-importer.cfg` configuration.
+
+Examples are `ansible-doc`, `ansible-lint`, and `flake8`.
+
+This parameter enables you to drive this configuration.
+
+2+| For {HubNameMain} to connect to LDAP directly; the following variables must be configured. 
```

Comment:
Should this go into a separate table?
Accessibility guidelines don't recommend multi-column table cells.